include/seqan3/alignment/aligned_sequence/aligned_sequence_concept.hpp

```diff
@@ -272,4 +285,97 @@ inline typename seq_type::iterator erase_gap(seq_type & seq, typename seq_type::
 }
 //!\}
 
+namespace detail
+{
+
+/*!
```

Comment:
please move the brief onto this line. This is done throughout the code base and should be consistent, no?
Same goes for all the other brief comments.